UnitsNet/CustomCode/Wrappers/Pressure.Wrapper.cs

```diff
@@ -0,0 +1,194 @@
+﻿// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnitsNet.CustomCode.Units;
+using UnitsNet.Units;
+
+namespace UnitsNet.CustomCode.Wrappers
+{
+    /// <summary>
+    ///     Pressure tied to a real-world reference, allowing conversion between references.
+    ///     <list type="bullet">
+    ///         <item>
+    ///             <description>Absolute is referenced to true vacuum.</description>
+    ///         </item>
+    ///         <item>
+    ///             <description>Gauge references the local atmospheric pressure.</description>
+    ///         </item>
+    ///         <item>
+    ///             <description>Vacuum is the negative of the gauge.</description>
+    ///         </item>
+    ///     </list>
+    /// </summary>
+    public struct ReferencePressure
```

Comment:
I would rename this file to `ReferencePressure.cs`. The `Pressure.xxx.cs` naming convention is typically used for partial types, when two files both hold code for a single type `Pressure`.